src/index.js

```diff
@@ -1 +1,118 @@
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const formidable = require('formidable');
+
+const PORT = process.env.PORT || 3000;
+const jsonFile = './src/expence.json';
+
+const server = new http.Server();
+
+server.on('request', (req, res) => {
+  const normilizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const pathName = normilizedUrl.pathname.slice(1) || 'index.html';
+
+  if (pathName === 'index.html') {
+    res.setHeader('Content-Type', 'text/html');
+
+    const readData = fs.createReadStream(`./src/public/${pathName}`);
+
+    readData.on('error', () => {
+      res.statusCode = '400';
```

Comment:
You are setting the status code to a string, but it should be a number. Use 400 instead of '400'.